packages/playground/data-liberation/src/import/WP_Topological_Sorter.php

```diff
@@ -70,10 +59,58 @@ class WP_Topological_Sorter {
 	 */
 	protected $current_item = 0;
 
+	const ENTITY_TYPES = array(
+		'comment'      => 1,
+		'comment_meta' => 2,
+		'post'         => 3,
+		'post_meta'    => 4,
+		'term'         => 5,
+	);
+
+	private $mapped_pre_filters = array(
+		// Name of the filter, and the number of arguments it accepts.
+		'wxr_importer_pre_process_comment' => 2,
```

Comment:
Just noting any filter names I left in are temporary and will need rethinking, e.g. should we have data source-specific filters such as `wxr_`? Or should we filter at the entity level? Also, we'll need filters for plugin authors to process their custom blocks, e.g. WP Bakery serializes markup as base64 and they'll need to decode it to enable rewriting URLs in there. I only mean this comment to inform, I don't think any of that should hold up the topo sort.